v3/pkg/application/webview_window_darwin.go

```diff
@@ -443,30 +443,28 @@ void windowSetHideTitle(void* nsWindow, bool hideTitle) {
 }
 
 // Set Window use toolbar
-void windowSetUseToolbar(void* nsWindow, bool useToolbar, int toolbarStyle) {
+void windowSetUseToolbar(void* nsWindow, bool useToolbar) {
```

Comment:
The signature of this method has changed because the style configuration is now done through the `windowSetToolbarStyle` method, as per the comment at line 467.